specs/protocol/README.md

```diff
@@ -136,16 +146,32 @@ An `Offering` is used by the PFI to describe a currency pair they have to _offer
 | `min`                    | [`DecimalString`](#decimalstring)       | N        | minimum amount required to use this payment method.                                                                                                 |
 | `max`                    | [`DecimalString`](#decimalstring)       | N        | maximum amount allowed when using this payment method.                                                                                              |
 
+> [!IMPORTANT]
+> `kind` should be considered as a unique identifier that is used to _identify_ / _specify_ an individual payment method
 
 > [!IMPORTANT]
 > the presence of `min` or `max` in a payment method takes precedence over `min` or `max` defined at the `PaymentDetails` level.
 
-
 > [!IMPORTANT]
-> `kind` should be considered as a unique identifier that is used to _identify_ / _specify_ an individual payment method
+> If `requiredPaymentDetails` is omitted, then any RFQs submitted for the respective offering must also omit `paymentDetails`. 
+
+#### `PayoutMethod`
+| field                     | data type                               | required | description                                                                                                                                         |
+| ------------------------- | --------------------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `kind`                    | string                                  | Y        | Type of payment method (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`, `SQUARE_PAY`). considered to be a unique identifier                                   |
+| `estimatedSettlementTime` | uint                                    | Y        | estimated time taken to settle an order. expressed in seconds                                                                                       |
```

Comment:
an Alice might want to fully understand what this field means (when her wallet renders this value), that:
- this estimate is for once a tbdex Order is submitted / payin has been received | settled?
- this estimate excludes all the other steps prior to tbdex exchange, such as establishing customer relationship by kyc onboarding

Reply:
ah. i guess this is mostly explained in [these lines](https://github.com/TBD54566975/tbdex/pull/269/commits/0614cf575b465581137908312c6e37c31864391c#diff-766e6d8ea5c6a7fcd675d0b65a33e80bc4aa6be519e9c2ff968ea952abf59947R173-R174) below